src/managers/TimestampManager.cpp

```diff
@@ -551,7 +664,12 @@ TimestampManager::decompressNextValue(std::vector<int> schemeVals, BitReader *bi
     }
 
     // Read number of bits corresponding to the found control bit
-    *currentVal = readBits(bitReader, currentSchemeVal);
+    if(valueIsSigned){
+        *currentVal = readBitsSigned(bitReader, currentSchemeVal);
+    } else {
+        *currentVal = readBits(bitReader, currentSchemeVal);
+    }
+//    std::cout << "CURRENT VAL: " << *currentVal << " bits to read: " << currentSchemeVal << std::endl;

```

Comment:
Check if any of this code is not used any longer after huffman encoding is implemented

Reply:
@teisvh

Reply:
Hvis kommentaren er sat til "decompressNextValue"-funktionen, så bliver den stadig brugt i delta-delta dekomprimeringen. Tænker, at vi måske skal beholde al delta-delta for nu, så vi kan sammenligne med dd i paper?

Reply:
@agneborn98